src/app/shared.module.ts

```diff
@@ -0,0 +1,14 @@
+import { CommonModule } from "@angular/common";
+import { NgModule } from "@angular/core";
+import { ContentLoaderComponent } from "./components/content-loader/content-loader.component";
+import { XchPipe } from "./pipes/xch.pipe";
+
+const COMPONENTS = [ContentLoaderComponent]
+const PIPES = [XchPipe]
```

Comment:
I think lowercase `pipes` would be enough.
The linter is generally based on Airbnb code style (with caveats) and they don't recommend using ALL_CAPS for local consts https://github.com/airbnb/javascript#naming--uppercase